central/notifiers/cscc/cscc.go

```diff
@@ -3,29 +3,33 @@ package cscc
 import (
 	"context"
 	"fmt"
-	"sort"
-	"strings"
-	"time"
+	"net/http"
 
+	secretmanager "cloud.google.com/go/secretmanager/apiv1"
+	securitycenter "cloud.google.com/go/securitycenter/apiv1"
+	"cloud.google.com/go/securitycenter/apiv1/securitycenterpb"
+	"github.com/googleapis/gax-go/v2"
 	"github.com/pkg/errors"
 	clusterDatastore "github.com/stackrox/rox/central/cluster/datastore"
-	"github.com/stackrox/rox/central/notifiers/cscc/client"
-	"github.com/stackrox/rox/central/notifiers/cscc/findings"
 	notifierUtils "github.com/stackrox/rox/central/notifiers/utils"
 	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/administration/events/codes"
+	adminOption "github.com/stackrox/rox/pkg/administration/events/option"
 	"github.com/stackrox/rox/pkg/cryptoutils/cryptocodec"
 	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/httputil/proxy"
 	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/notifiers"
-	"github.com/stackrox/rox/pkg/protoconv"
-	"github.com/stackrox/rox/pkg/retry"
 	"github.com/stackrox/rox/pkg/sac"
 	"github.com/stackrox/rox/pkg/sac/resources"
 	"github.com/stackrox/rox/pkg/utils"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc"
 )
 
 var (
-	log = logging.LoggerForModule()
+	log = logging.LoggerForModule(adminOption.EnableAdministrationEvents())
```

Comment:
Did we want to collect events for notifiers/registries only, or for backup integrations as well?

Reply:
Currently only notifiers, if we want backup integrations as well we'd have to move this.

But, this is probably something we'd like to do with the deprecation of the health status API, so not something we want to do right away.